apps/condo/pages/property/[id]/index.tsx

```diff
@@ -102,35 +255,86 @@ export const PropertyPageContent = ({ property, role }) => {
-                    <PropertyInfoPanel title={TicketsInWorkTitle} message={property.ticketsInWork}  type='warning' large />
-                </Col>
-                <Col flex={0}>
-                    <PropertyInfoPanel title={AreaTitle} message={property.area ? property.area : UnknownValueTitle } />
-                </Col>
-                <Col flex={0}>
-                    <PropertyInfoPanel title={YearOfConstructionTitle} message={yearOfConstructionCardLabel} />
-                </Col>
-                <Col flex={0} >
-                    <PropertyInfoPanel title={UnitsCountTitle} message={property.unitsCount} large />
-                </Col>
-                <Col flex={0} >
-                    <PropertyInfoPanel title={UninhabitedUnitsCountTitle} message={property.uninhabitedUnitsCount} large />
-                </Col>
-            </Row>
+            {propertyBankAccountPage ? (
+                <Row
+                    gutter={PROPERTY_PAGE_CONTENT_ROW_INFO_BLOCK_GUTTER}
+                    style={PROPERTY_PAGE_CONTENT_ROW_INFO_BLOCK_STYLE}
+                >
+                    <Col xl={12} md={24} sm={24} xs={24}>
+                        <PropertyListWrapper>
+                            <List title={PropertyInformationTitle} dataSource={propertyInfoDataSource} />
+                            <List title={TicketsTitle} dataSource={propertyTicketDataSource} />
+                        </PropertyListWrapper>
+                    </Col>
+                    <Col xl={12} md={24} sm={24} xs={24}>
+                        <Card>
+                            <PropertyCardContent>
+                                <Space direction='vertical' size={12}>
+                                    <Typography.Title level={4}>{PropertyReportTitle}</Typography.Title>
+                                    <Typography.Text>{PropertyReportDescription}</Typography.Text>
+                                </Space>
+                                <PropertyReportCardBottomWrapper>
+                                    <Space direction='vertical' size={12}>
+                                        <Button type='sberDefaultGradient'>
+                                            {SetupReportTitle}
+                                        </Button>
+                                        <Button
+                                            type='sberDefaultGradient'
+                                            secondary
+                                            onClick={createBankAccountRequestCallback}
+                                            loading={createBankAccountRequestLoading}
+                                        >
+                                            {BecomeSberClientTitle}
+                                        </Button>
+                                    </Space>
+                                    <ImageWrapper>
+                                        <Image src='/property-empty-report.png' preview={false} />
```

Comment:
move source to constant

Reply:
Into what constants? I can't find any examples of same situations